modules/restore-add-master.adoc

```diff
@@ -48,6 +51,8 @@ ETCD_INITIAL_CLUSTER_STATE="existing"'
 Member  1e42c7070decd39 added to cluster 6e25bab1bb556673
 Starting etcd..
 ----
+<1> The IP address of another etcd member that is already active. This is _not_ the IP address of the member you are adding.
```

Comment:
To avoid some issues that the ISG would have with tense and implicit words, I might just say:
> The IP address of an active etcd member.

Or:
> The IP address of an active etcd member.
[IMPORTANT
====
You must specify the IP address of the active etcd member first.
====